docs/content/configuration-language.mdx

```diff
@@ -580,6 +582,12 @@ This particular use of referencing relations on related objects is defining a tr
 
 This can be used to indicate that **viewers** of a **folders** are **viewers** of all **documents** in that **folder**.
 
+:::caution
+Note that <ProductName format={ProductNameFormat.LongForm}/> does not allow the referenced relation (the word after `from`, also called the tupleset) to be referencing another relation or allow non-concrete types (type bound public access (`<object_type>:*`) or usersets (`<object_type>#<relation>`)) in its type restrictions and will throw a validation error when attempting to call `WriteAuthorizationModel`.
+
+Note: In the deprecated 1.0 schema version, the validation error will be thrown at the time of evaluation (`Check`, etc..).
```

Comment:
Should we have link to the migration document so that the reader understands what is schema 1.0?

Reply:
Done